tests/python/relay/aot/test_crt_aot.py

```diff
@@ -364,5 +364,27 @@ def test_byoc_utvm(use_calculated_workspaces):
     compile_and_run(mod, input_list, output_list, use_calculated_workspaces)
 
 
+def test_quant_mobilenet_tfl():
```

Comment:
could you clarify which part of the PR this is testing?

Reply:
This is testing  #8062. Should I explicitly mention the issue in a comment?

Reply:
yes, it would be great to explain the thing we are trying to test using mobilenet, and if possible, contrive a testcase not based on data from the internet.

Reply:
can you also do this for the test case below?